#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_grml_version_file_hook.dpatch by <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Copy /etc/grml_version to initrd

@DPATCH@

Index: b/hooks/live
===================================================================
--- a/hooks/live	2011-07-24 22:07:22.000000000 +0200
+++ b/hooks/live	2011-07-24 22:07:55.000000000 +0200
@@ -40,6 +40,12 @@
 	done
 fi
 
+# Grml version information:
+if [ -r /etc/grml_version ]
+then
+       cp /etc/grml_version "${DESTDIR}"/etc
+fi
+
 # Handling live-boot
 
 [ "${QUIET}" ] || echo -n " "core
